### PR TITLE
Added create date function

### PR DESCRIPTION
**Pull request by SaarCohen93**
According to the issue "[Sending message] (optional) automate the “create date” field so that the user don’t have to enter it manually"
create function that return today's date.
Although, I think there is a way to use the DB logs to do that, need to check out how its done.

### CODE REVIEW

**Comment by lmilbaum** [comment]
@SaarCohen93 First line in the first comment should be:
'fix <issue number>'

File: app.py [hunk 1]
```diff
@@ -1,18 +1,15 @@
 #!/usr/bin/env python3
+from datetime import date
```
Comment (lmilbaum) [inline app.py:2]: did you run flake8?

File: app.py [hunk 2]
```diff
@@ -1,18 +1,15 @@
...
-
-
 @app.route('/contact_us')
 def contact_us():
     return render_template('/contact_us.html')
-
-
 if __name__ == '__main__':
-    app.run(host='0.0.0.0')
+    app.run(debug=True)
```
Comment (lmilbaum) [inline app.py:14]: why did you remove the host argument?

File: homepage/HomePage.py [hunk 3]
```diff
@@ -0,0 +1,7 @@
+from flask import Flask, render_template
```
Comment (lmilbaum) [inline homepage/HomePage.py:1]: I don't understand what this file is for? How is it used by the tiger app?

File: homepage/templates/home.html [hunk 4]
```diff
@@ -0,0 +1,7 @@
+<!DOCTYPE html>
```
Comment (lmilbaum) [inline homepage/templates/home.html:1]: How is this file related?

**Review by lmilbaum — CHANGES_REQUESTED** [review]

**Comment by lmilbaum** [comment]
@SaarCohen93 You should NOT work on the master branch!!!

File: homepage/HomePage.py (hunk 3 above)
Comment (SaarCohen93) [reply homepage/HomePage.py:1]: My mistake, pushed the wrong folder :(

File: homepage/templates/home.html (hunk 4 above)
Comment (SaarCohen93) [reply homepage/templates/home.html:1]: same, will be fixed right away

**Comment by SaarCohen93** [comment]
@lioramilbaum hmmm ok, so where should i push my changes?
